src/components/gas/GasSpeedButton.js

```diff
@@ -260,8 +260,8 @@ const GasSpeedButton = ({
       }
     },
     [
-      openCustomGasSheet,
       gasFeeParamsBySpeed,
+      setShouldOpenCustomGasSheet,
```

Comment:
oh I missed this:
`setShouldOpenCustomGasSheet` can be removed from all of the dependency lists in this file

Reply:
there are more places it is added the the dependencies than just here